packages/utilities/moment-timezone-include/src/index.js

```diff
@@ -0,0 +1,19 @@
+import webpack from 'webpack';
+import writeNewTimezoneData from './writeNewTimezoneData';
+import replaceOriginalTimezoneData from './replaceOriginalTimezoneData';
+
+function MomentTimezoneInclude(options = {}) {
```

Comment:
🤔 could fat arrow be used here?

Reply:
Needs to be a constructor and
>Arrow function expressions are ill suited as methods, and they cannot be used as constructors.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions

(ikr TIL)